r_examples/r_serving_with_plumber/iris.csv

```diff
@@ -0,0 +1,151 @@
+Sepal.Length,Sepal.Width,Petal.Length,Petal.Width,Species
```

Comment:
Please remove all three copies of the Iris dataset and only use the AWS-hosted verison.

Reply:
I can do that. 

FYI, the iris.csv that I included is the same one provided in this example: 
https://github.com/aws/amazon-sagemaker-examples/tree/master/r_examples/r_byo_r_algo_hpo

Would you like me to submit a separate PR to update that example?

Reply:
I see, yes a separate PR to remove the committed dataset from the other example would be welcome.

Reply:
I just realized that I can't remove the duplicate copies of the xgb.model file since it is copied into the docker image at build time. I could rewrite each container to map it at runtime from another sagemaker example folder, but it seems like that would make the repo more brittle in the long-run. Another contributor might change the single model and unintentional break the other examples.